static/js/utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Returns true if the value is either the boolean value
+ * 'true' or a string representation of 'true'
+ * 
+ * @param {string|boolean} value
+ * @returns {boolean}
+ */
+export function canonicalizeBoolean (value) {
+  if (typeof(value) === 'string') {
```

Comment:
super nit but I normally see typeof without the parenthesis (I didn't even know you could do it with parens lol). Do we have a preference between the two?

Reply:
I think i prefer it without